### DISCUSSION TITLE
3.0.1

### DISCUSSION THREAD

**Discussion by timmo001** [timmo001/system-bridge#1630] [General]
## ⬆️ Dependency updates

- Bump @mui/lab from 5.0.0-alpha.79 to 5.0.0-alpha.80 in /frontend @dependabot (#1624)
- Bump @mui/material from 5.6.3 to 5.6.4 in /frontend @dependabot (#1625)
- Bump zeroconf from 0.38.4 to 0.38.5 in /backend @dependabot (#1627)
- Bump pywin32 from 303 to 304 in /backend @dependabot (#1628)
- Bump @types/node from 17.0.30 to 17.0.31 in /frontend @dependabot (#1623)

<hr /><em>This discussion was created from the release <a href='https://github.com/timmo001/system-bridge/releases/tag/3.0.1'>3.0.1</a>.</em>

**Comment by boardwarp** [comment]
How do we get the API key for this release?

**Reply by timmo001** [reply]
Please check the documentation:

https://system-bridge.timmo.dev/docs/running

https://system-bridge.timmo.dev/docs/cli
